### PR TITLE
fix error format

### PR DESCRIPTION
**Pull request by cuileon**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: docs/guide-zh-CN/input-forms.md [hunk 1]
```diff
@@ -233,8 +233,8 @@ Pjax::end();
 
 #### 提交按钮和文件上传中的值
 
-在处理 [[https://github.com/jquery/jquery/issues/2321|files]] 和
-[[https://github.com/jquery/jquery/issues/2321|submit button values]] 
+在处理 (files)[https://github.com/jquery/jquery/issues/2321] 和
+(submit button values)[https://github.com/jquery/jquery/issues/2321]
```
Comment (cebe) [inline docs/guide-zh-CN/input-forms.md]: The brackets are in wrong order, link title should be in `[]` and URL in `()`. A good way to remember this is that `[]` are valid characters in a URL, while `()` are not, so URL must be wrapped in `()`.

**Comment by cebe** [comment]
btw, if you add a `translators.json` file, your names will be listed in the PDF as translation authors:

https://github.com/yiisoft/yii2/blob/d07e286dbdc2d5990942bb71ee49698a0051091d/docs/guide-ru/translators.json

**Comment by cuileon** [comment]
> btw, if you add a `translators.json` file, your names will be listed in the PDF as translation authors:
> 
> https://github.com/yiisoft/yii2/blob/d07e286dbdc2d5990942bb71ee49698a0051091d/docs/guide-ru/translators.json

I will now add the translation authors according to the example you gave.

File: docs/guide-zh-CN/input-forms.md (hunk 1 above)
Comment (cuileon) [reply docs/guide-zh-CN/input-forms.md]: I'm sorry, I will fix it.

**Comment by machour** [comment]
@cuileon Is this PR finished and ready to merge?

**Comment by samdark** [comment]
Looks like so.

**Comment by samdark** [comment]
Thank you @cuileon.

**Pull request merged**